.github/workflows/ci.yml

```diff
@@ -0,0 +1,136 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+# For details on the macOS enviornment see https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md
+#
+# We try to align the versions of tools with what is already available on the image to reduce the time it takes to run our CI.
+# At the time of writing these are:
```

Comment:
May be more informative to say the current date

```suggestion
# At the time of writing (October 2020) these are:
```

Reply:
I like  it